index.d.ts

```diff
@@ -26,6 +26,7 @@ declare namespace React {
   function cloneElement(element: any, props?: Props, ...children: any[]): any
 
   var Children: ReactChildren;
+  const Fragment: any;
```

Comment:
I wonder if there is any reason why we wouldn't want to declare this as `const` rather than `var`.  
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f2982c175477516d52bdc40191d7829d0531ba35/types/react/index.d.ts#L269 uses `const`.

Reply:
Is it fine to use `any`?